docs/core-concepts/core-sdk/cli/using-ci-with-cli.md

```diff
@@ -0,0 +1,58 @@
+# Using .NET Core SDK and tools in Continuous Integration
+
+## Overview
+This document outlines the usage of .NET Core SDK and its tools on the build server. In general, on a CI build server, 
```

Comment:
CI build server -> Continuous Integration (CI) build server
through some way? Can't you just say "you want to automate the installation process"?
if at all possible -> if possible